cfy_manager/main.py

```diff
@@ -1407,5 +1405,16 @@ def main():
     os.umask(current_umask)
 
 
+def _ensure_root():
+    skip_root_check = '--skip-root-check'
+    if skip_root_check in sys.argv:
+        sys.argv.remove(skip_root_check)
+    else:
+        if os.geteuid() != 0:
+            sys.exit(subprocess.call(
+                ['sudo'] + sys.argv + [skip_root_check]))
+
+
 if __name__ == '__main__':
+    _ensure_root()
```

Comment:
ah, this is not the right place, unfortunately. Just calling `cfy_manager` goes by setuptools entry points (cf. setup.py), so it calls the `main` function directly, and doesn't go through this `if __name__` block at all. This block is, for all that I can see, entirely useless. So this PR doesn't actually run the check :P

Reply:
Well, that was clearly obvious to anyone who was paying attention (not me yesterday afternoon, it seems)- nice spot, thanks!

I'll rip out that last block, yes- as you point out, it's useless. As I point out, it can serve as a cunning trap.